.gitignore

```diff
@@ -52,6 +52,12 @@ bld/
 # Visual Studio 2017 auto generated files
 Generated\ Files/
 
+# scenario intermediate directories
+src/scenarios/*/pub
+src/scenarios/*/app
+src/scenarios/*/bin
+src/scenarios/*/traces
```

Comment:
add obj folder?

Reply:
do we have anything that actually makes an obj folder in this place? I haven't seen one yet.

Reply:
oops  I meant bin and obj in the app folder and they are already covered, my mistake.